src/_flow/reporter.js

```diff
@@ -20,8 +20,8 @@ declare interface Reporter {
 }
 
 declare class Sender {
-  append(span: Span): SenderResponse;
-  flush(): SenderResponse;
+  append(span: Span): Promise<SenderResponse>;
+  flush(): Promise<SenderResponse>;
```

Comment:
how will the Promise class get translated to Node 0.10? Is there some backfill implementation that babel provides?

Reply:
There is a babel polyfill.  I'm checking if Babel will implement it for us for Node 0.10 and 0.12. 🤞

Reply:
😌 I have so much appreciation for Babel.